libpoolprotocols/PoolManager.cpp

```diff
@@ -49,6 +49,11 @@ PoolManager::PoolManager(PoolClient * client, Farm &farm, MinerType const & mine
 		dev::setThreadName("main");
 		cnote << "Disconnected from " + m_connections[m_activeConnectionIdx].Host() << p_client->ActiveEndPoint();
 
+		// Clear queue of submission times as we won't get any further response for them (if any left)
+		while (m_submit_times.size()) {
```

Comment:
m_submit_times.clear() would be more concise.

Reply:
Nevermind... Not a list but a queue.  don't think the queue class has a clear method.